### ISSUE TITLE
Migrate Genie to 5.0.0

### ISSUE THREAD

**Issue by averyniceday** [knowledgesystems/pipelines-scrum#934]
## Done Condition (What do we need? Why do we need it? Keep this is small as possible!)
GENIE portals and databases are migrated to 5.0.0
Have curation externally convert files first and test inside triage before importing and releasing 

- [x] Clone and freeze DB - redirect production to cloned db
- [x] Migrate original DB - delete fusion studies
- [ ] re-import SV studies
- [ ] build new importers (pointing to production spreadsheet) 
- [ ] Update property files
- [ ] Coordinate with Gaofei to rollout SV to actual portal

**Comment by averyniceday** [comment]
Confirmed with GaoFei that they are waiting for curation to finish converting files.

Curation has confirmed that files are almost done being converted to SV (probably by 9/13)
In the meantime we have created `cgds_genie_release_5_0_0` in the genie rds node. We are dumping current `cgds_genie` - will reload and remove all studies with fusion profiles and then re-import.

Important things to note:
There is a potential genie release (13.3) later this week - may need to re-convert depending on when files are provided.
The conversion script that is currently deployed to run as part of the importer does not work for GENIE files - curation is almost done updating edge cases but this should be tested in more detail. Might be better to import actual data_sv files first before relying on conversion script.
